src/script/common/c_content.cpp

```diff
@@ -1564,6 +1585,127 @@ ToolCapabilities read_tool_capabilities(
 	return toolcap;
 }
 
+/******************************************************************************/
+PointabilityType read_pointability_type(lua_State *L, int index)
+{
+	if (lua_isboolean(L, index)) {
+		if (lua_toboolean(L, index))
+			return PointabilityType::POINTABLE;
+		else
+			return PointabilityType::POINTABLE_NOT;
+	} else if (lua_isstring(L, index)) {
```

Comment:
FYI directly using `luaL_checkstring` would be slightly shorter here

Reply:
Fixed